docs/README.md

```diff
@@ -641,6 +643,46 @@ With proper concurrency logic in place, the latest run might have been cancelled
           workflow: my-workflow.yml
 ```
 
+### github-deployment-create and github-deployment-status-update
+
+These actions create a [GitHub deployment](https://docs.github.com/en/rest/deployments/deployments) and allow updating its status. That can be useful to track progression on a workflow pipeline.
+
+Sample usage:
+
+```yaml
+permissions:
+  deployments: write # This is required for deployment statuses management
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Deployment
+        id: create-deployment
+        uses: Alfresco/alfresco-build-tools/.github/actions/github-deployment-create@ref
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          environment: my_gh_environment
+          state: in_progress
+
+      - name: Other Steps
+
+      - name: Update Deployment State to failure
+        if: always() && failure() && steps.create-deployment.outcome == 'success'
```

Comment:
`always() && failure()` isn't equivalent to just `failure()`?

Reply:
sadly, no

Reply:
actually the doc at https://docs.github.com/en/actions/learn-github-actions/expressions#always recommend `if: ${{ !cancelled() }}`

`always() && failure()` is needed when you want a step in another job to be called even if previous job failed.

Anyway it's the readme, I can update

Reply:
For the use case I documented you're right that it's not needed (and I tested) so I updated :+1: